src/test/java/com/jwplayer/southpaw/SouthpawByteArraySetEndToEndTest.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2018 Longtail Ad Solutions (DBA JW Player)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jwplayer.southpaw;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jwplayer.southpaw.json.DenormalizedRecord;
+import com.jwplayer.southpaw.record.BaseRecord;
+import com.jwplayer.southpaw.topic.BaseTopic;
+import com.jwplayer.southpaw.util.ByteArray;
+import com.jwplayer.southpaw.util.FileHelper;
+import com.jwplayer.southpaw.util.KafkaTestServer;
+
+import scala.Tuple2;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+
+public class SouthpawByteArraySetEndToEndTest {
+    private static final String CONFIG_PATH = "test-resources/ByteArraySetTest/config.yaml.tmpl";
```

Comment:
the standard convention is to throw these into a `resources` directory within the `test` directory. These files will get thrown into the root classpath within the test context